src/main/java/org/sonar/plugins/stash/issue/collector/SonarQubeCollector.java

```diff
@@ -40,132 +23,22 @@ private SonarQubeCollector() {
    * Create issue report according to issue list generated during SonarQube
    * analysis.
    */
-  public static SonarQubeIssuesReport extractIssueReport(ProjectIssues projectIssues, InputFileCache inputFileCache, File projectBaseDir) {
-    SonarQubeIssuesReport result = new SonarQubeIssuesReport();
+  public static List<Issue> extractIssueReport(ProjectIssues projectIssues, IssuePathResolver issuePathResolver) {
+    List<Issue> result = new ArrayList<>();
 
     for (Issue issue : projectIssues.issues()) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)